### PR TITLE
update various yarn lock and package json files to resolve security vulnerabilities

### PR DESCRIPTION
**Pull request by dasormeter**
Upgrading yarn lock files throughout the repo to address a few security advisories and generally bring things to the modern age

image

### CODE REVIEW

File: language-support/ts/codegen/tests/ts/yarn.lock [hunk 1]
```diff
@@ -1,4314 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (cocreature) [inline language-support/ts/codegen/tests/ts/yarn.lock:1]: Why did you delete this? afaik this is used when running the tests outside of Bazel.

Comment (dasormeter) [reply language-support/ts/codegen/tests/ts/yarn.lock:1]: sorry not done yet -- switched to draft, still checking a few things

Comment (dasormeter) [reply language-support/ts/codegen/tests/ts/yarn.lock:1]: hiya @cocreature long discussion with @shayne-fletcher about this today.  conclusion is it is not really needed as that folder has some auto-magic yarn installin' that happens in a temp folder so the lock file provides little benefit, and the yarn utilities don't really work standalone because of the dynamic codegen dependencies that come in

Comment (dasormeter) [reply language-support/ts/codegen/tests/ts/yarn.lock:1]: so i think we are okay to delete it unless any objections @hurryabit ?

Comment (hurryabit) [reply language-support/ts/codegen/tests/ts/yarn.lock:1]: No objections.
